README.md

```diff
@@ -1,3 +1,280 @@
 # sf-functions-python
 
-Python support for Salesforce Functions.
+Python support for [Salesforce Functions](https://developer.salesforce.com/docs/platform/functions/overview).
+
+> Note: This feature is in beta and has been released early so we can collect feedback. It may contain significant problems, undergo major changes, or be discontinued. The use of this feature is governed by the Salesforce.com Program Agreement.
+
+---
+
+# Getting Started with Python for Functions
+
+## Prerequisites
+
+> Any install commands that follow assume a Mac OSX system with Homebrew available.  If you’re on another OS you’ll have to click through the links to get alternative install instructions.
+
+### [Install Python 3](https://www.python.org/downloads/)
+
+```sh
+brew install python3
+```
+
+The installed Python version should be at least `3.10.6` or higher  
```

Comment:
Where does the `3.10.6` come from? The latest 3.10.x is 3.10.9 (released just over a week ago), however any 3.10.x would probably do.

Reply:
i just took the versions off my local setup but i can change it to `3.10`